examples/config.py

```diff
@@ -8,30 +13,49 @@ def setup_examples():
         import infermedica_api
     except ImportError:
         import sys
-        import os
 
         sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
         import infermedica_api
 
-    # !!! ENTER YOUR CREDENTIALS HERE !!!
-    infermedica_api.configure({
-        'app_id': 'YOUR_APP_ID',
-        'app_key': 'YOUR_APP_KEY',
-        'dev_mode': True  # Use only during development on production remove this parameter
+    # !!! SET YOUR CREDENTIALS AS ENVIRONMENTAL VARIABLES "APP_ID" & "APP_KEY" OR SET THEM HERE !!!
+    app_id = os.getenv('APP_ID', 'YOUR_APP_ID')
+    app_key = os.getenv('APP_KEY', 'YOUR_APP_KEY')
+
+    # Prepare API v3 connector as default one
+    infermedica_api.configure(**{
+        'app_id': app_id,
+        'app_key': app_key,
+        'dev_mode': True  # Use only during development or testing/staging, on production remove this parameter
     })
 
-    import logging
+    # Prepare API v2 connector under 'v2' alias
+    infermedica_api.configure(**{
+        'alias': 'v2',
+        'api_connector': 'ModelAPIv2Connector',
```

Comment:
Isn't `v2/v3` alias enough? What's the use for selecting other connector?

Reply:
You can put anything you want in the alias, it does not imply usage of any connector, a more advanced use case:
```
infermedica_api.configure(**{
    'alias': 'v2-en',
    'api_connector': 'ModelAPIv2Connector',
    'model': 'infermedica-en',
    'app_id': 'XXX',
    'app_key': 'XXX'
})

infermedica_api.configure(**{
    'alias': 'v2-de',
    'api_connector': 'ModelAPIv2Connector',
    'model': 'infermedica-de',
    'app_id': 'XXX',
    'app_key': 'XXX'
})


infermedica_api.configure(**{
    'alias': 'v3-en',
    'api_connector': 'APIv3Connector',
    'model': 'infermedica-en',
    'app_id': 'XXX',
    'app_key': 'XXX'
})

...
api_v2_en = infermedica_api.get_api('v2-en')
api_v2_de = infermedica_api.get_api('v2-de')
api_v3_en = infermedica_api.get_api('v3-en')
```